Gemfile

```diff
@@ -22,6 +22,10 @@ gem "listen"
 gem "puma"
 gem "deface"
 
+if ["v4.1", "v4.2"].include?(solidus_branch)
+  gem "concurrent-ruby", "< 1.3.5"
```

Comment:
It seems it is fixed in Rails 7.1
Can we just bump the Rails version in this File L20 to` >= 7.1`?

Reply:
Solidus 4.1/4.2 only allow Rails 7.0.x